From 6d248766c5073c07a0d2a0899cdb78a26e821198 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 30 May 2026 10:35:54 +0800
Subject: [PATCH 4/4] AOSCOS: use PKGCONFIGDIR for .pc installation path
 variable

Autobuild uses PKGDIR to represent the package root, which is obviously
a conflict.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 0ff4e2c..f873179 100644
--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ GLEW_DEST ?= /usr/local
 BINDIR    ?= $(GLEW_DEST)/bin
 LIBDIR    ?= $(GLEW_DEST)/lib
 INCDIR    ?= $(GLEW_DEST)/include/GL
-PKGDIR    ?= $(GLEW_DEST)/lib/pkgconfig
+PKGCONFIGDIR    ?= $(GLEW_DEST)/lib/pkgconfig
 
 ifneq ($(GLEW_NO_GLU), -DGLEW_NO_GLU)
 LIBGLU = glu
@@ -239,9 +239,9 @@ install.include:
 	$(INSTALL) -m 0644 include/GL/eglew.h "$(DESTDIR)$(INCDIR)/"
 
 install.pkgconfig: glew.pc
-	$(INSTALL) -d -m 0755 "$(DESTDIR)$(PKGDIR)"
-	$(INSTALL) -d -m 0755 "$(DESTDIR)$(PKGDIR)"
-	$(INSTALL) -m 0644 glew.pc "$(DESTDIR)$(PKGDIR)/"
+	$(INSTALL) -d -m 0755 "$(DESTDIR)$(PKGCONFIGDIR)"
+	$(INSTALL) -d -m 0755 "$(DESTDIR)$(PKGCONFIGDIR)"
+	$(INSTALL) -m 0644 glew.pc "$(DESTDIR)$(PKGCONFIGDIR)/"
 
 uninstall:
 	$(RM) "$(DESTDIR)$(INCDIR)/eglew.h"
-- 
2.52.0

